components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1245,6 +1284,7 @@ def __init__(self, argument, index, args, argc, descriptorProvider,
             isClamp=argument.clamp,
             isMember="Variadic" if argument.variadic else False,
             isAutoRooted=type_needs_auto_root(argument.type),
+            needsRootingInto="arg_root" if type_conversion_needs_root(argument.type) else False,
```

Comment:
The current typed array conversion API requires an existing stack root, so I wanted to also pass the name of stack root here or `False` if it's not needed.

In general I don't feel too good about it, since it's only used with argument conversion code and the `arg_root` name is duplicated anyway.

This seemed like a general mechanism that could be supported (e.g. union member conversions use a stack root before the conversion), but I think we can avoid introducing this by attempting the conversion on unrooted object and only root before the call (like with custom auto rooter), but this'd warrant changes to typed arrays API in rust-mozjs.

Should I do that?